Makefile

```diff
@@ -183,17 +184,19 @@ lua-test:
 .PHONY: e2e-test
 e2e-test:
 	echo "Granting permissions to ingress-nginx e2e service account..."
-	kubectl create serviceaccount ingress-nginx-e2e || true
+	kubectl create serviceaccount ingress-nginx-e2e --context $(KUBE_CONTEXT) || true
 	kubectl create clusterrolebinding permissive-binding \
 		--clusterrole=cluster-admin \
 		--user=admin \
 		--user=kubelet \
+		--context $(KUBE_CONTEXT) \
 		--serviceaccount=default:ingress-nginx-e2e || true
 
 	kubectl run --rm -i --tty \
 		--attach \
 		--restart=Never \
 		--generator=run-pod/v1 \
+		--context $(KUBE_CONTEXT) \
```

Comment:
I think this will not age well - we can easily forget to add `--context` later when we decide to create anothe resource.